contracts/Colony.sol

```diff
@@ -78,6 +78,9 @@ contract Colony is ColonyStorage, PatriciaTreeProofs {
     IColonyNetwork colonyNetwork = IColonyNetwork(colonyNetworkAddress);
     uint256 rootLocalSkill = colonyNetwork.getSkillCount();
     initialiseDomain(rootLocalSkill);
+
+    // Set initial colony reward inverse amount
```

Comment:
Shouldn't there be a require over this? Or a call to the `setRewardInverse` function?

Reply:
One alternative would be to set a default value in the `initialiseColony ` function and allow people to update it later. I like that as well because it keeps the `initialiseColony` interface small and so makes me happy from an aesthetic perspective.

Reply:
Bump. Someone could set a 0 here and leave it unchanged.